modules/openxBidAdapter.js

```diff
@@ -300,6 +318,10 @@ function appendUserIdsToQueryParams(queryParams, userIds) {
           break;
         case 'lipb':
           queryParams[key] = userIdObjectOrValue.lipbid;
+          if (Array.isArray(userIdObjectOrValue.segments) && userIdObjectOrValue.segments.length > 0) {
+            const liveIntentSegments = 'liveintent:' + userIdObjectOrValue.segments.join('|')
```

Comment:
does it make more sense to modify the liveintent adapter to write their segments to ortb2 than to modify your adapter to read them from here?

Reply:
I agree, it certainly does, but modifying the liveintent adapter is out of my scope for this task and I don't feel confident I can do it properly at this time, I just started working on Prebid.js. We can perhaps ask them to consider making this change.

Reply:
@melgenek fyi

Please feel free to reach out to us on Prebid slack or join a taxonomy committee meeting and we can discuss how you can make it so all adapters can easily absorb your segments.